filip/models/ngsi_v2/base.py

```diff
@@ -19,18 +19,12 @@
 class Http(BaseModel):
     """
     Model for notification and registrations sent or retrieved via HTTP
-    """
-    url: Union[AnyHttpUrl, str] = Field(
-        description="URL referencing the service to be invoked when a "
-                    "notification is generated. An NGSIv2 compliant server "
-                    "must support the http URL schema. Other schemas could "
-                    "also be supported."
-    )
 
-    @field_validator('url')
-    @classmethod
-    def check_url(cls, value):
-        return validate_http_url(url=value)
+    url: It references the service to be invoked when a notification is
+    generated. An NGSIv2 compliant server must support the http URL
+    schema. Other schemas could also be supported.
+    """
+    url: AnyHttpUrl
```

Comment:
Description is missing

Reply:
I could not find a way to add description for it, because it is of type `Annotated`. Do you know any?

Reply:
Now I bring the `Field` back